hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/troubleshooting_s3a.md

```diff
@@ -1216,15 +1217,16 @@ java.lang.SecurityException: Instruction file not found for S3 object with bucke
     at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:95)
     at org.apache.hadoop.fs.FsShell.main(FsShell.java:390)
 ```
+
```

Comment:
unrelated?

Reply:
because codeblocks in markdown typically also have a new line after them some IDE's complain about this.